extensions/blocks/podcast-player/style.scss

```diff
@@ -227,6 +238,11 @@ $player-float-background: $light-gray-200;
 		}
 	}
 
+	// needs slightly higher specificity to override some theme's link styles
```

Comment:
There are no CSS documentation guidelines for WordPress that I'm aware of, so let's reuse the PHP & JS guidelines? (Mainly starting with a capitalized letter and ending on a period.)
https://make.wordpress.org/core/handbook/best-practices/inline-documentation-standards/php/#5-1-single-line-comments

Reply:
Done! Updated my comments to start with a capital letter and end on a period.